### PR TITLE
Add mechanism to inject ENV variables to wizard

### PR DESCRIPTION
**Pull request by mariogmz**
- Prepares the app for injecting env vars from the k8s manifests
- It generates the config.js file programmatically by grabbing a base config file, replacing the server's env variables and writing a new file for the client js
- Now the app is visible through the genesys console

https://apps.usw2.pure.cloud/directory/#/apps/f40f480e-a554-4318-9ac5-00763db53745
image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@ node_modules/
 _scratch/
 **/__test_full_config.js
 validation.log
+wizard/config/config.js
```
Comment (lsurasani) [inline .gitignore:5]: why was this added to `.gitignore`?

**Review by lsurasani — APPROVED** [review]

**Review by ozmar-salesloft — APPROVED** [review]
